test/close-test.js

```diff
@@ -1,22 +1,9 @@
 const test = require('tape')
-const testCommon = require('abstract-leveldown/testCommon')
-const leveldown = require('..')
-const abstract = require('abstract-leveldown/abstract/close-test')
+const testCommon = require('./common')
+const abstract = require('abstract-leveldown/test/close-test')
 
-module.exports.setUp = function () {
-  test('setUp', testCommon.setUp)
-}
+test('setUp', testCommon.setUp)
 
-module.exports.close = abstract.close
+abstract.close(testCommon.factory, test, testCommon)
 
-module.exports.tearDown = function () {
-  test('tearDown', testCommon.tearDown)
-}
-
-module.exports.all = function (leveldown) {
-  module.exports.setUp()
-  module.exports.close(leveldown, test, testCommon)
-  module.exports.tearDown()
-}
-
-module.exports.all(leveldown)
+test('tearDown', testCommon.tearDown)
```

Comment:
Cleaned up the `module.exports`. Never understood why we need to export methods here.

Reply:
Could be useful to reuse the leveldown test suite in rocksdb etc? idk